contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -6846,6 +6848,8 @@ post_process_column_definition(TSqlParser::Column_definitionContext *ctx, PLtsql
 static void
 post_process_table_constraint(TSqlParser::Table_constraintContext *ctx, PLtsql_stmt_execsql *stmt, TSqlParser::Ddl_statementContext *baseCtx)
 {
+	if (ctx->UNIQUE())
+		rewritten_query_fragment.emplace(std::make_pair(ctx->UNIQUE()->getSymbol()->getStopIndex()+1 , std::make_pair("", " NULLS NOT DISTINCT")));
```

Comment:
Is there any other places need to be replaced ?
Will this rewrite resued with other sql syntax ? ( like alter )

Reply:
`CREATE TABLE`, `ALTER TABLE ... ADD CONSTRAINT`, `CREATE INDEX` only these 3 need rewriting.
This function will handle create table and alter table rewritting.

Reply:
only other SQL where unique constraint is possible is inline index, which is not supported in babelfish.
I've added proper unsupported error for inline indexes